pkg/osd_cluster_upgrader/upgrader.go

```diff
@@ -407,6 +429,25 @@ func ControlPlaneUpgraded(c client.Client, scaler scaler.Scaler, metricsClient m
 			isCompleted = true
 		}
 	}
+
+	var upgradeStartTime, controlPlaneCompleteTime *metav1.Time
+	for _, uc := range upgradeConfig.Status.History {
```

Comment:
Can we base this on ClusterVersion startedTime for version? Seems more correct if we our timeout is for controlplane